recipes/colcon-cmake/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "colcon-cmake" %}
+{% set version = "0.2.14" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/colcon/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 5f87370765b10cccfca2117e32cf194361703a478d88b34037f94ca76c1d3189
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```

For those that become `noarch: python`:
```suggestion
    - python >=3.6
```